.gitignore

```diff
@@ -2,3 +2,4 @@
 /.idea/*
 .vscode
 .DS_Store
+.history
```

Comment:
```suggestion
```

Reply:
Thank you for the fast response. I have to ask: `.history` is the default path where VSCode stores its local history. Is there any reason, why you don't want that in here?

Reply:
Hi @mgmgwi, I use VSCode and don't have that file maybe it is related to another type of project.

Reply:
Sorry - my bad, I have the Local History extension running. I fixed this.